test/type_prop/roi_align.cpp

```diff
@@ -0,0 +1,30 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+TEST(type_prop_layers, roi_align_basic_shape_inference)
```

Comment:
Need much more unit tests for the shape inference:
1. Dynamic batch dimension in the "data" tensor.
2. Dynamic channels dimension in the "data" tensor.
3. Check that input tensors shape mismatch is handled correctly.

Reply:
I've added some more tests but I don't think the first one that you suggested is needed. The batch dimension of the data input does not contribute to the output shape.

Reply:
For the first case I actually wanted to have dynamic batch size of the rois tensor.